.circleci/config.yml

```diff
@@ -42,7 +36,7 @@ jobs:
           name: Generate config
           command: |
             export INT_MATRIX=$(jq -c 'if (.all) then
-              (["detection", "fr", "classification", "workflow"])
+              (["workflow"])
```

Comment:
We should be able to remove complexity in a follow-up now that there is only the `workflow` option here.